tests/test_pastebin_plugin.py

```diff
@@ -152,21 +152,14 @@ def fake_wait_window(success_dialog):
 
 
 def test_paste_error_handling(monkeypatch, caplog, mocker, tabmanager, filetab, dont_run_in_thread):
-    monkeypatch.setattr(pastebin_module, "DPASTE_URL", "ThisIsNotValidUrlStart://wat")
-    mock = mocker.patch("porcupine.plugins.pastebin.errordialog")
+    mocker.patch("tkinter.messagebox.showerror")
 
-    tabmanager.select(filetab)
-    get_main_window().event_generate("<<Menubar:Pastebin/dpaste.com>>")
-    get_main_window().update()
+    get_main_window().tk.call("tk", "busy", "hold", get_main_window())
+    pastebin_module.pasting_done_callback(pastebin_module.DPaste(), tkinter.Toplevel(), False, "")
```

Comment:
I don't really like how a test about error handling has to know how exactly `pasting_done_callback` gets called, and how exactly the main window is made busy.

Is there a reason to rewrite this part of the test? Was there a problem with invoking `"<<Menubar:Pastebin/dpaste.com>>"`?